dev/com.ibm.ws.microprofile.faulttolerance.cdi/src/com/ibm/ws/microprofile/faulttolerance/cdi/config/AsynchronousConfig.java

```diff
@@ -44,7 +45,7 @@ public void validate() {
         if (method != null) {
             Class<?> originalMethodReturnType = getAnnotatedMethod().getReturnType();
             if (!(Future.class.isAssignableFrom(originalMethodReturnType))) {
-                throw new FaultToleranceDefinitionException(Tr.formatMessage(tc, "asynchronous.method.not.returning.future.CWMFT5001E", method));
+                throw new FaultToleranceDefinitionException(Tr.formatMessage(tc, "asynchronous.method.not.returning.future.CWMFT5001E", FTConstants.formatMethod(method)));
```

Comment:
If you really want to use the same constants class then move it into the spi bundle